packages/ui/src/stories/formkit/File.stories.ts

```diff
@@ -0,0 +1,24 @@
+import { StoryFn } from '@storybook/vue3'
+import * as types from '@vuestic/formkit'
+
+export default {
+  title: 'Formkit Integration/File',
+}
+
+export const Default: StoryFn = () => ({
+  setup () {
+    return {
+      types,
+    }
+  },
+  template: `
+    <div class="w-1/5 grid gap-6">
+      <FormKit
+        :type="types.file"
```

Comment:
Can't we use the same syntax as formkit?

Adding types imports looks messy to me.

```
<FormKit
    type="file"
```